### PR TITLE
Add com.locomalito.abbayedesmorts

### PR DESCRIPTION
**Pull request by Rampoina**
Flatpak for the free implementation [1] of the video game l'Abbaye des morts [2]

[1] https://github.com/nevat/abbayedesmorts-gpl
[2] https://www.locomalito.com/abbaye_des_morts.php

### CODE REVIEW

File: com.locomalito.abbayedesmorts.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+  "app-id": "com.locomalito.abbayedesmorts",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "18.08",
```
Comment (bilelmoussaoui) [inline com.locomalito.abbayedesmorts.json]: Any reason for not using 19.08?

File: com.locomalito.abbayedesmorts.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  ],
+  "modules": [
+      {
+          "name": "abbayedesmorts",
+          "buildsystem": "simple",
+          "build-commands": [
+              "make PREFIX=/app",
+              "make install PREFIX=/app",
+              "install -Dm644 com.locomalito.abbayedesmorts.desktop /app/share/applications/com.locomalito.abbayedesmorts.desktop",
+              "install -Dm644 com.locomalito.abbayedesmorts.appdata.xml /app/share/metainfo/com.locomalito.abbayedesmorts.appdata.xml"
```
Comment (bilelmoussaoui) [inline com.locomalito.abbayedesmorts.json]:
```suggestion
              "install -Dm644 com.locomalito.abbayedesmorts.metainfo.xml /app/share/metainfo/com.locomalito.abbayedesmorts.metainfo.xml"
```

File: com.locomalito.abbayedesmorts.desktop [hunk 3]
```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+Name=Abbaye des Morts
+Comment=Indie platform game by Locomalito
+Comment[ca]=Joc indie de plataformes de Locomalito
+Comment[es]=Juego indie de plataformas de Locomalito
+Exec=abbayev2
+Icon=abbaye
```
Comment (bilelmoussaoui) [inline com.locomalito.abbayedesmorts.desktop]:
```suggestion
Icon=com.locomalito.abbayedesmorts
```

Comment (bilelmoussaoui) [reply com.locomalito.abbayedesmorts.desktop]: You will need to install the icon with the correct name too.

File: com.locomalito.abbayedesmorts.appdata.xml [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+      <li>23 screens to explore.</li>
+      <li>Riddles and hints to find items.</li>
+      <li>Around 30 minutes of game length (once fully mastered).</li>
+  </ul>
+  </description>
+  <developer_name>LocoMalito</developer_name>
+  <url type="homepage">https://www.locomalito.com</url>
+  <url type="help">https://github.com/nevat/abbayedesmorts-gpl/blob/master/ReadMe.md</url>
+  <launchable type="desktop-id">com.locomalito.abbayedesmorts.desktop</launchable>
+  <provides>
```
Comment (bilelmoussaoui) [inline com.locomalito.abbayedesmorts.appdata.xml]: Not needed.

File: com.locomalito.abbayedesmorts.appdata.xml [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+      </p>
+      <ul> Features
+      <li>Look and feel of a ZX Spectrum game.</li>
+      <li>23 screens to explore.</li>
+      <li>Riddles and hints to find items.</li>
+      <li>Around 30 minutes of game length (once fully mastered).</li>
+  </ul>
+  </description>
+  <developer_name>LocoMalito</developer_name>
+  <url type="homepage">https://www.locomalito.com</url>
```
Comment (bilelmoussaoui) [inline com.locomalito.abbayedesmorts.appdata.xml]: I would add a bugtracker url here.

File: com.locomalito.abbayedesmorts.appdata.xml [hunk 6]
```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component type="desktop">
```
Comment (bilelmoussaoui) [inline com.locomalito.abbayedesmorts.appdata.xml]:
```suggestion
<component type="desktop-application">
```

File: com.locomalito.abbayedesmorts.appdata.xml [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+  <id>com.locomalito.abbayedesmorts</id>
+  <metadata_license>CC-BY-3.0</metadata_license>
+  <project_license>GPLv3</project_license>
+  <content_rating type="oars-1.1" />
```
Comment (bilelmoussaoui) [inline com.locomalito.abbayedesmorts.appdata.xml]: Not sure if that's a good rating of the game.

File: com.locomalito.abbayedesmorts.json (hunk 1 above)
Comment (Rampoina) [reply com.locomalito.abbayedesmorts.json]: Hello, this is my first time creating a flatpak. No special reason to use 18.08, the game that I used as a template used this version. I would have to test if it works with 19.08

File: com.locomalito.abbayedesmorts.appdata.xml (hunk 7 above)
Comment (Rampoina) [reply com.locomalito.abbayedesmorts.appdata.xml]: My fault, using this https://hughsie.github.io/oars/generate.html  I come up with:
Although I'm unsure if the character dying when touching a monster would be mild or moderate.

  <content_rating type="oars-1.0">
    <content_attribute id="violence-cartoon">moderate</content_attribute>
    <content_attribute id="violence-fantasy">moderate</content_attribute>
  </content_rating>

File: com.locomalito.abbayedesmorts.json (hunk 1 above)
Comment (Rampoina) [reply com.locomalito.abbayedesmorts.json]: It does work. Updating runtime, thanks.

**Comment by LeandroStanger** [comment]
bot, build com.locomalito.abbayedesmorts

File: com.locomalito.abbayedesmorts.desktop (hunk 3 above)
Comment (Rampoina) [reply com.locomalito.abbayedesmorts.desktop]: fixed in 0cc2245e0ae4d3ce05733da505675804c9915d74

File: com.locomalito.abbayedesmorts.json (hunk 1 above)
Comment (Rampoina) [reply com.locomalito.abbayedesmorts.json]: updated in ee96dc1fee4fc891b121e85445d215561ce2075a

File: com.locomalito.abbayedesmorts.appdata.xml (hunk 7 above)
Comment (Rampoina) [reply com.locomalito.abbayedesmorts.appdata.xml]: fixed in be7b9b3be258120a6750cb58061a09da1acf6c2f

File: com.locomalito.abbayedesmorts.appdata.xml (hunk 5 above)
Comment (Rampoina) [reply com.locomalito.abbayedesmorts.appdata.xml]: Shouldn't that point to the project's homepage? I could add the github one here instead
https://github.com/nevat/abbayedesmorts-gpl

**Comment by Rampoina** [comment]
bot, build com.locomalito.abbayedesmorts

**Comment by Rampoina** [comment]
bot, build com.locomalito.abbayedesmorts

File: com.locomalito.abbayedesmorts.json (hunk 2 above)
Comment (Rampoina) [reply com.locomalito.abbayedesmorts.json]: fixed in 0bab1b7cc2b73133fc013f36453fc031bf989819 and e18979c6e6964f793ec08865c93ae7b80502e586

**Comment by barthalion** [comment]
Can you contact upstream team about your submission? I don't expect an answer but just to be safe.

**Pull request closed without merging**